include/gatekeeper_net.h

```diff
@@ -312,6 +311,11 @@ struct net_config {
 	unsigned             mailbox_mem_cache_size;
 	unsigned             mailbox_burst_size;
 
+	/* Number of attempts to wait for a link to come up. */
+	uint8_t              num_attempts_link_get;
```

Comment:
I think the comment should give a little bit more information, maybe: `/* Number of attempts to wait for Gatekeeper links to come up during initialization. */`, split into multiple lines as needed.

Reply:
Also I think this can be an `unsigned int`, but `ipv6_default_hop_limits` should stay a `uint8_t` since that is the size it needs to be to be put into a packet header.